sw/device/sca/aes_serial.c

```diff
@@ -51,7 +53,22 @@ static aes_cfg_t aes_cfg;
  */
 static void aes_serial_set_key(const uint8_t *key, size_t key_len) {
   SS_CHECK(key_len == kAesKeyLength);
-  aes_key_put(key, /*key_share1=*/(uint8_t[kAesKeyLength]){0}, aes_cfg.key_len);
+  dif_aes_transaction_t transaction = {
+      .key_len = kDifAesKey128,
+      .masking = kDifAesMaskingForceZero,
```

Comment:
`kDifAesMaskingForceZero` -> `kDifAesMaskingInternalPrng`

With `kDifAesMaskingForceZero` you basically disable the masking.

Reply:
Yes, lib/aes.c doesn't seem to touch this bit.

Reply:
Done.